whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java

```diff
@@ -69,7 +70,7 @@ public MailMessage parse(final MimeMessage message, final UpdateContext updateCo
         if (deliveryDate != null && deliveryDate.length > 0 && deliveryDate[0].length() > 0) {
             messageBuilder.date(deliveryDate[0]);
         } else {
-            messageBuilder.date(DATE_FORMAT.format(ZonedDateTime.now()));
+            messageBuilder.date(DATE_FORMAT.format(ZonedDateTime.now(ZoneOffset.UTC)));
```

Comment:
Could ```dateTimeProvider.getCurrentDateTimeUtc()``` be used instead?

Reply:
Updated to use `dateTimeProvider.getCurrentZonedDateTime()`